lib/router/router.js

```diff
@@ -109,6 +126,15 @@ p.getInfo = function(req) {
     }
   }
 
+  // forward original image if no operation on image and browser support image format
+  if (routeInfo.imageSteps.length === 0 && this.options.ifNoOperation && this.options.ifNoOperation.justForward === true) {
```

Comment:
I think that I missed the useOriginal query. I think it works for me. I am checking. I have large original that I need access not so often times. I use mostly resized versions, but I would like both in same endpoint.